src/Modules/Settings/Dnn.PersonaBar.Security/Security.Web/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "set NODE_ENV=production&&webpack -p",
-    "webpack": "webpack-dev-server -d --port 8080 --hot --inline --content-base dist/ --history-api-fallback",
+    "webpack": "webpack-dev-server -d --host 0.0.0.0 --port 8080 --hot --inline --content-base dist/ --history-api-fallback",
```

Comment:
I added this parameter here so that the dev-server can be accessed from virtual machine (running VMWare on Mac). [This answer](https://apple.stackexchange.com/a/180636/253832) explains why it's necessary.